app/templates/components/event-card.hbs

```diff
@@ -17,7 +17,7 @@
           @src={{if this.event.thumbnailImageUrl this.event.thumbnailImageUrl this.event.originalImageUrl}} />
       </a>
     {{/unless}}
-    <a class="{{if this.device.isMobile '' 'card-height'}} main content" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.identifier}}">
+    <a class="{{if (or (not this.minHeight) this.device.isMobile) '' 'card-height'}} main content" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.identifier}}">
```

Comment:
use instead ```{{unless (or this.fullHeight this.device.isMobile) 'card-height'}}```

Reply:
I was doing this but there was a lint error 
"Using {{unless}} in combination with other helpers should be avoided."

Reply:
so use ```{{if (not (or this.fullHeight this.device.isMobile)) 'card-height'}}```